code/modules/projectiles/ammunition.dm

```diff
@@ -65,15 +65,12 @@
 	if(G)
 		put_residue_on(G)
 		var/mob/living/carbon/human/H = G.get_recursive_loc_of_type(/mob/living/carbon/human)
```

Comment:
It's not quite clear at a glance that it stands for bodypart, and it kind of relies on knowing what the function returns. Even then I'm not really sure what it returns. An object? A define? The next line (after the conditional) implies it's not just bodyparts, it's equipment slots, but it can also be an organ tag??